tests/ToolInfoTest.php

```diff
@@ -21,13 +21,87 @@
  */
 final class ToolInfoTest extends \PHPUnit_Framework_TestCase
 {
+    const PHPUNIT_BIN = '/../vendor/bin/phpunit';
+
+    protected function setUp()
+    {
+        foreach ($this->provideScriptName() as $testArguments) {
+            $symlink = current($testArguments);
+            if ($symlink === self::PHPUNIT_BIN or is_link(__DIR__.$symlink)) {
```

Comment:
For the sake of homogeneity and safety.
First, as the project currently never uses `or`.
Second, as most devs cannot properly understand that difference - even if you would, then new contributor probably wont when he will be touching this code.